### PR TITLE
Committing for teammate to review my assignment.

### PR DESCRIPTION
**Pull request by Laurand**
Hello Crystal, could you please review my assignment. Thank you.

Added an AssignmentController with stories.
-Add a new assignment.
-Change the name of the assignment.
-Delete the assignment.

### CODE REVIEW

File: cst438_gradebook/src/main/java/com/cst438/controllers/AssignmentController.java
Comment (Crystal-cloud-max) [inline cst438_gradebook/src/main/java/com/cst438/controllers/AssignmentController.java:1]: I see you have no errors in this added file with 3 new functions of new assignments, change a name, and delete an assignment.

File: cst438_gradebook/src/test/java/com/cst438/JunitTestGradebook.java
Comment (Crystal-cloud-max) [inline cst438_gradebook/src/test/java/com/cst438/JunitTestGradebook.java:1]: I think the code is understandable and clean. I see you added functions to delete and add assignments,

**Review by Crystal-cloud-max — COMMENTED** [review]
I reviewed your 2 files AssignmentController and JunitTesGradebook the code looks awesome that will run with no errors.

File: cst438_gradebook/src/test/java/com/cst438/JunitTestGradebook.java
Comment (Laurand) [reply cst438_gradebook/src/test/java/com/cst438/JunitTestGradebook.java:1]: Hi Crystal, thank you for taking the time to review my commits and the nice feedback. I thought it was very interesting assignment and I have learned a lot, including Getting, posting, testing and collaborating.

**Pull request merged**